### PR TITLE
Added 'pack image tag' param to 'az acr pack build'

### PR DESCRIPTION
**Pull request by dorfire**

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/acr/_params.py [hunk 1]
```diff
@@ -145,6 +145,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('registry_name', options_list=['--registry', '-r'])
         c.argument('image_name', options_list=['--image', '-t'], help="The name and tag of the image using the format: '-t repo/image:tag'.")
         c.argument('builder', options_list=['--builder', '-b'], help="The name and tag of a Buildpack builder image.")
+        c.argument('pack_image_tag', options_list=['--pack-image-tag'], help="The tag of the 'pack' runner image ('mcr.microsoft.com/oryx/pack').", is_preview=True)
```
Comment (northtyphoon) [inline src/azure-cli/azure/cli/command_modules/acr/_params.py]: Can you call out the default is 'stable'?

Comment (dorfire) [reply src/azure-cli/azure/cli/command_modules/acr/_params.py]: it should mention that when a customer passes `--help`.

**Comment by yonzhan** [comment]
@qwordy please take a look.

**Comment by qwordy** [comment]
> 
> 
> @qwordy please take a look.

OK

File: src/azure-cli/azure/cli/command_modules/acr/_params.py (hunk 1 above)
Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/acr/_params.py]: Is this conversation resolved?

**Comment by qwordy** [comment]
In the above commit,
1. Solve conflict.
2. CI has problems. Merge latest dev branch to contain the fix of CI.

**Comment by dorfire** [comment]
> In the above commit,
> 
> 1. Solve conflict.
> 2. CI has problems. Merge latest dev branch to contain the fix of CI.

Thanks - merged from latest dev.

**Pull request merged**